package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-native-safe-area-context",
-  "version": "1.0.2",
+  "version": "2.0.0",
```

Comment:
Leave the version like this I'll take care of updating that when doing a release.